Sources/ISDBTibs/TibsBuilder.swift

```diff
@@ -406,10 +406,12 @@ extension TibsBuilder {
 
   /// The default sdk path to use on Darwin (on other platforms, returns nil).
```

Comment:
Comment is now out of date 😅. Would it hurt to use the environment variable as a default for all non-macos platforms?

Reply:
Sure, we should be able to do that as Linux will not set `SDKROOT` by default.  IIRC, unlike Windows and Darwin, Linux will effectively treat `-sdk` as `--sysroot`.

Reply:
Sure, we should be able to do that as Linux will not set `SDKROOT` by default.  IIRC, unlike Windows and Darwin, Linux will effectively treat `-sdk` as `--sysroot`.